CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - FATE code generator improvements.
 - Bug fix: Handle qualified constructors in patterns.
 - Bug fix: Allow switching also on negative numbers.
+- Bug fix: Fixed ACI encoder to handle `-` unary operator
```

Comment:
Wrong place, as this is unreleased.

Reply:
Also, seems like the changelog is out of date, should be fixed as part of the next point release.

Reply:
I will update the other changes in changelog as a separate PR

Reply:
This is intentional... We update the changelog on every release, not on each commit...

Reply:
I mixed up different repos. The changelog isn't out of date, my bad. But still the location of the entry was incorrect.